### ISSUE TITLE
Do a new release

### ISSUE THREAD

**Issue by rufuspollock** [okfn/ckanclient-deprecated#8]
With the new datastore code in here plus code for filestore we should do a new release.

Release command:

```
python setup.py register sdist upload
```

**Comment by rufuspollock** [comment]
@seanh assigning to your for review and input.

**Comment by seanh** [comment]
@rgrp I think it's ready for push to PyPI (which I don't have permission to do). I've updated the README and CHANGELOG, incremented the version number in `__init__.py` for PyPI, and created the tag for the release.

The README was a bit of a mess with some parts (e.g. datastore client) documented in the README file and other parts (ckan catalog client) documented in `__init__.py`. I've moved all README and CHANGELOG text out of `__init__.py` into the README and CHANGELOG files so they show up nicely on GitHub. PyPI should probably be updated with links to the git repo for the documentation and changelog.
